controller/controller.go

```diff
@@ -181,7 +181,7 @@ func (c *Controller) Detach(detachRequest k8sresources.FlexVolumeDetachRequest)
 	var response k8sresources.FlexVolumeResponse
 	c.logger.Debug("", logs.Args{{"request", detachRequest}})
 
-	if c.isVersion15() {
+	if detachRequest.Version == k8sresources.KubernetesVersion_1_5 {
```

Comment:
detachRequest.Version  ? I don't see any PR in the ubiquity repo for that.
What do I miss?

In addition, I don't see check of the version in the install script. is it already in place or different PR?

Reply:
the detach request version is determined by the cli.go
its not in ubiquity repo